src/code.sh

```
@@ -0,0 +1,59 @@
+    -a /home/dnanexus/genome_lib/${lib_dir}/ctat_genome_lib_build_dir/ref_genome.fa -g /home/dnanexus/genome_lib/${lib_dir}/ctat_genome_lib_build_dir/ref_annot.gtf \
+    -b //home/dnanexus/arriba/database/blacklist_hg38_GRCh38_v*.tsv.gz -k /home/dnanexus/arriba/database/known_fusions_hg38_GRCh38_v*.tsv.gz -t /home/dnanexus/arriba/database/known_fusions_hg38_GRCh38_v*.tsv.gz -p /home/dnanexus/arriba/database/protein_domains_hg38_GRCh38_v*.gff3
+
+# Move output files with error handling
+for file in "${sample_name}_fusions.tsv" "${sample_name}_fusions.discarded.tsv"; do
+    if [ ! -f "/home/dnanexus/out/$file" ]; then
+        echo "Error: Expected output file not found: $file"
+        exit 1
+    fi
+done
+
+if ! mv /home/dnanexus/out/${sample_name}_fusions.tsv /home/dnanexus/out/output_fusions_files || \
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)